recon_test_pack/run_test_listmode_recon.sh

```diff
@@ -0,0 +1,141 @@
+#! /bin/sh
+# A script to check to see if reconstruction of simulated data gives the expected result.
+#
+#  Copyright (C) 2011 - 2011-01-14, Hammersmith Imanet Ltd
+#  Copyright (C) 2011-07-01 - 2011, Kris Thielemans
+#  Copyright (C) 2014, University College London
+#  This file is part of STIR.
+#
+#  This file is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+#  This file is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  See STIR/LICENSE.txt for details
+#      
+# Author Nikos Efthimiou
```

Comment:
well you know, quite a lot of it is copy-paste...

Reply:
;)